WeatherMood/ViewController/BaseViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  BaseViewController.swift
+//  WeatherMood
+//
+//  Created by 이지원 on 2021/07/22.
+//
+
+import UIKit
+import RxSwift
+
+class BaseViewController: UIViewController {
+    // MARK: - Coordinator
+    weak var coordinator: MainCoordinator?
+
+    // MARK: - Properties
+    var viewModel: ViewModelType?
+
+    // MARK: - Rx
+    var disposeBag: DisposeBag = DisposeBag()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .systemBackground
```

Comment:
```suggestion
        view.backgroundColor = .systemBackground
```

self는 이니셜라이저나, 클로저에서만 쓰는걸로 해요! 
부스트코스리뷰받을때 굳이필요없는 self는 안쓰는게좋다라고 리뷰받았었어요. 저도 그게좋다라고 생각합니다.